src/services/user.ts

```diff
@@ -298,6 +316,16 @@ function updateSocialLink(platform: Platform, link: string) {
   return http.put('profiles/' + platform.toLowerCase(), body);
 }
 
+type ProfileFieldUpdate = {
+  Field: string;
+  Value: string;
+  Label: string;
+};
+
+const requestInfoUpdate = async (updatedFields: ProfileFieldUpdate[]) => {
+  await http.post('profiles/update/', updatedFields);
+};
```

Comment:
This will technically change the behavior of this function, since it will now return the result of the post request (whereas previously it didn't). That shouldn't matter though, since the return value is ignored in the only place where it's called.

Technically, if we're after simplicity, we could simplify it even further:
```suggestion
const requestInfoUpdate = (updatedFields: ProfileFieldUpdate[]) => 
  http.post('profiles/update/', updatedFields);
```

Reply:
Yes, that looks good.